things/api/http/requests_test.go

```diff
@@ -58,16 +58,16 @@ func TestAddThingReqValidation(t *testing.T) {
 
 func TestUpdateThingReqValidation(t *testing.T) {
 	key := uuid.NewV4().String()
-	id := uuid.NewV4().String()
+	id := uint64(1)
 
 	cases := map[string]struct {
 		thing things.Thing
-		id    string
+		id    uint64
 		key   string
 		err   error
 	}{
 		"valid thing update request": {thing, id, key, nil},
-		"non-uuid thing ID":          {thing, wrong, key, things.ErrNotFound},
+		"invalid thing ID":           {thing, 0, key, things.ErrNotFound},
```

Comment:
Why `wrong` constant is not used here, and it is in many other places in tests? It would increase code self-documentation.